python/dataproc_templates/pubsublite/README.md

```diff
@@ -0,0 +1,73 @@
+
+## Arguments
+
+* `pubsublite.to.bq.input.subscription`: Cloud Storage location of the input files (format: `gs://bucket/...`)
+* `pubsublite.to.bq.write.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to append)
+* `pubsublite.to.bq.output.dataset`: BigQuery output dataset name
+* `pubsublite.to.bq.output.table`: BigQuery output table name
+* `pubsublite.to.bq.bucket.name`: Temporary bucket for the Spark BigQuery connector
+
+## Usage
+
+```
+$ python main.py --template PUBSUBLITETOBQ --help
+usage: main.py --template PUBSUBTOBQ [-h] \
+    --pubsublite.to.bq.input.subscription PUBSUBLITE.BIGQUERY.INPUT.SUBSCRIPTION \
+    --pubsublite.to.bq.project.id PUBSUBLITE.BIGQUERY.OUTPUT.PROJECT.ID \
+    --pubsublite.to.bq.output.dataset PUBSUBLITE.BIGQUERY.OUTPUT.DATASET \
+    --pubsublite.to.bq.output.table PUBSUBLITE.BIGQUERY.OUTPUT.TABLE \
+    --pubsublite.to.bq.bucket.name PUBSUBLITE.BIGQUERY.BUCKET.NAME \
+    [--pubsublite.to.bq.write.mode {overwrite,append,ignore,errorifexists}] \
+    --pubsublite.to.bq.checkpoint.location PUBSUBLITE.BIGQUERY.CHECKPOINT.LOCATION
+```
+
+## Required JAR files
+
+This template requires the [Pubsublite Spark SQL Streaming](https://central.sonatype.com/artifact/com.google.cloud/pubsublite-spark-sql-streaming/1.0.0) to read the streaming data from Pubsublite and [Spark BigQuery connector](https://cloud.google.com/dataproc-serverless/docs/guides/bigquery-connector-spark-example) for writing to BigQuery.
+
+## Example submission
+
+```
+export GCP_PROJECT=<project_id>
+export REGION=<region>
+export GCS_STAGING_LOCATION=<gcs-staging-bucket-name> 
+export JARS="gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar,gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
+./bin/start.sh \
+-- --template=PUBSUBLITETOBQ \
+--pubsublite.to.bq.input.subscription=<pubsublite input subscription> \
+--pubsublite.to.bq.project.id=<BQ project ID> \
+--pubsublite.to.bq.output.dataset=<BQ dataset name> \
+--pubsublite.to.bq.output.table=<BQ table name> \
+--pubsublite.to.bq.write.mode=<Output write mode> \
+--pubsublite.to.bq.bucket.name=<bucket name> \
+--pubsublite.to.bq.checkpoint.location=<checkpoint folder location>
+```
+
+
+## Configurable Parameters
+
+Following properties are available in commandline or [template.constants](../util/template_constants.py) file:
+
+```
+## Project that contains the input Pub/Sub subscription to be read
+pubsubliteinput.project.id=<pubsublite project id>
+## PubSublite subscription name
+pubsubliteinput.subscription=<pubsublite subscription>
+## Stream timeout, for how long the subscription will be read
+pubsublitetimeout.ms=60000
+## Streaming duration, how often wil writes to BQ be triggered
+pubsublitestreaming.duration.seconds=15
+## Number of streams that will read from Pub/Sub subscription in parallel
+pubsublitetotal.receivers=5
```

Comment:
Where is it used?

Reply:
Removed this parameter and executed the template and it ran successfully. So, removed this parameter -> [commit1](https://github.com/GoogleCloudPlatform/dataproc-templates/pull/658/commits/90477ae3a3561666fa24d39b0427348de969c06c) & [commit2](https://github.com/GoogleCloudPlatform/dataproc-templates/pull/658/commits/673feef0cdcdc813c06567a0ee922a0ef2e0de13)